.github/workflows/playwright.yml

```
@@ -4,6 +4,13 @@ on:
     branches: [ main, staging ]
   pull_request:
     branches: [ main, staging ]
+  workflow_run:
+    workflows:
+      - "FalkorDb/Falkordb: browser tests trigger on new tag release"
+    types:
+      - completed
+    repositories:
```

actionlint (1.7.4)

unexpected key "repositories" for "workflow_run" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)